buildbot/master/master.cfg

```diff
@@ -6,12 +6,10 @@ from buildbot.status.results import SUCCESS
 
 from passwords import HTTP_USERNAME, HTTP_PASSWORD, SLAVE_PASSWORD, CHANGE_PASSWORD
 from passwords import GITHUB_DOC_TOKEN, GITHUB_STATUS_TOKEN, HOMU_BUILDBOT_SECRET
-from passwords import AWS_ACCESS_KEY, AWS_SECRET_KEY
 from passwords import MINION_PUBLIC_KEY, MINION_PRIVATE_KEY
```

Comment:
This line is no longer needed as well. In addition, we should remove these 4 variables from the passwords.py file and the test pillars.